core/workspace_comment_svg.js

```diff
@@ -24,6 +24,7 @@ goog.require('Blockly.utils.dom');
 goog.require('Blockly.utils.object');
 goog.require('Blockly.utils.Rect');
 goog.require('Blockly.WorkspaceComment');
+goog.require('Blockly.WorkspaceCommentSvg.render');
```

Comment:
This was deliberately left out so that others can extend the rendering part of a workspace comment. Previously, BlockSvg had the same structure.
You can just require the WorkspaceCommentSvg.render in the playground.